### PR TITLE
Add user and admin buttons

### PR DESCRIPTION
**Pull request by jenniferzoey**
- Conditionally render add new place and edit place buttons on pages based on user and admin status
- Wrote tests for admin and user buttons on site

### CODE REVIEW

File: spec/features/admin_can_create_a_place.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    id: 3,
+    first_name: "Chris",
+    last_name: "Fine",
+    email: "Cf@hotmail.com",
+    password: "password",
+    role: "admin"
+  )}
+
+  scenario 'admin is able to add a new place' do
+    visit new_user_session_path
```
Comment (CassandraNewell) [inline spec/features/admin_can_create_a_place.rb]: Please replace this logic (for signing in) with a sign in method, as described in https://launchboston25.slack.com/archives/CGV84UWFP/p1563904635004700

Comment (CassandraNewell) [reply spec/features/admin_can_create_a_place.rb]: do this for every one of these tests where your test currently has a user sign in

File: spec/features/admin_can_edit_a_place.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    visit new_user_session_path
+    fill_in 'Email', with: user3.email
+    fill_in 'Password', with: user3.password
+    click_button 'Log in'
+
+    visit "/places/#{place1.id}"
+    expect(page).to have_content("Edit")
+  end
+  scenario 'admin is able to edit a place they did create' do
```
Comment (CassandraNewell) [inline spec/features/admin_can_edit_a_place.rb]: Please add a space here, after line 16, and delete line 57! Prettiness counts :)

Comment (CassandraNewell) [reply spec/features/admin_can_edit_a_place.rb]: er, space == empty line

File: spec/features/admin_can_create_a_place.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    click_button 'Log in'
+
+    expect(page).to have_link("Add New Place")
+    expect(page).to have_current_path("/")
+
+    click_link 'Add New Place'
+    expect(page).to have_current_path("/places/new")
+  end
+
+  scenario 'unauthenticated non-admin user is not able to add a new place' do
```
Comment (CassandraNewell) [inline spec/features/admin_can_create_a_place.rb]: This is redundant with spec/features/user_can_create_new_place.rb -- it's okay to delete this one

File: spec/features/admin_can_create_a_place.rb (hunk 1 above)
Comment (jenniferzoey) [reply spec/features/admin_can_create_a_place.rb]: Fixed this. Thank you! :)

**Pull request merged**